BnbnavNetClient/Models/Player.cs

```diff
@@ -120,81 +130,130 @@ public void HandlePlayerMovedEvent(PlayerMoved evt)
         var newZ = evt.Z;
 
         // ReSharper disable CompareOfFloatsByEqualityOperator
```

Comment:
Is there a way to disable just for the next line?

Reply:
I don't know haha, this was actually there from before. Might be worth checking out though.

Reply:
disable once has a funny consequence :grin: